disnake/asset.py

```diff
@@ -340,6 +340,15 @@ def _from_avatar_decoration(cls, state: AnyState, avatar_decoration_asset: str)
             animated=animated,
         )
 
+    @classmethod
+    def _from_nameplate(cls, state: AnyState, nameplate_asset: str, animated: bool = True) -> Self:
+        return cls(
+            state,
+            url=f"{cls.BASE}/{nameplate_asset}/asset.{'webm' if animated else 'png'}",
```

Comment:
It looks like this isn't the url that discord uses. They're using https://cdn.discordapp.com/assets/collectibles/nameplates/nameplates/twilight/asset.webm

Reply:
https://github.com/DisnakeDev/disnake/pull/1303/commits/6ff7ba788a6b7f9b0d05c15426fbc222d44ad19b